tools/docker/tests/test_flows.sh

```diff
@@ -178,6 +178,39 @@ test_client_steering_mandate() {
         'grep -i -q "ACK_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
 }
 
+test_client_steering_dummy() {
+    status "test client steering dummy"
+    check_error=0
+
+    dbg "Connect dummy STA to wlan0"
+    check docker exec -it repeater1 sh -c \
+        'echo "AP-STA-CONNECTED 11:22:33:44:55:66" > /tmp/$USER/beerocks/wlan0/EVENT'
+
+    dbg "Send steer request "
+    eval send_bml_command "steer_client \"11:22:33:44:55:66 aa:bb:cc:00:00:20\"" $redirect
+    sleep 1
+
+    dbg "Confirming Client Steering Request message was received - mandate"
+    check docker exec -it repeater1 sh -c \
+        'grep -i -q "Got steer request" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
+
+    dbg "Confirming BTM Report message was received"
+    check docker exec -it gateway sh -c \
+        'grep -i -q "CLIENT_STEERING_BTM_REPORT_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_controller.log'
+
+    dbg "Confirming ACK message was received"
+    check docker exec -it repeater1 sh -c \
+        'grep -i -q "ACK_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
+
+    dbg "Disconnect dummy STA from wlan0"
+    check docker exec -it repeater1 sh -c \
+        'echo "AP-STA-DISCONNECTED 11:22:33:44:55:66" > /tmp/$USER/beerocks/wlan0/EVENT'
+    dbg "Connect dummy STA to wlan2"
+    check docker exec -it repeater1 sh -c \
+        'echo "AP-STA-CONNECTED 11:22:33:44:55:66" > /tmp/$USER/beerocks/wlan2/EVENT'
```

Comment:
The test should also verify that the corresponding messages are received and handled by the client task, i.e. verify log messages ["sta ... disconnected after successful steering"](https://github.com/prplfoundation/prplMesh/blob/9c196ee8789bd954857aec9e61650a1e91c74864/controller/src/beerocks/master/tasks/client_steering_task.cpp#L223) and ["steering successful for sta ..."](https://github.com/prplfoundation/prplMesh/blob/9c196ee8789bd954857aec9e61650a1e91c74864/controller/src/beerocks/master/tasks/client_steering_task.cpp#L220).

Reply:
Only "steering successful for sta " could be added now.
STA_DISCONNECTED is not sent by ACTION_CONTROL_CLIENT_DISCONNECTED_NOTIFICATION.
Added TODO.